### PR TITLE
Rebuild to update base image for security vulns

### PR DESCRIPTION
**Pull request by mfds**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,4 @@ Image variants tagged with 12-expocli also include:
  - 2021-11-16 -- Security fixes and latest expo-cli 4.13.0
  - 2021-11-06 -- Update to Alpine 3.14 base image
  - 2021-09-07 -- Security fixes and latest expo-cli 4.11.0
+ - 2022-01-17 -- Rebuild to update base image for security vulns
```
Comment (chriskurzeja) [inline README.md:26]: Needs to be at the top of the list - it's descending order

**Review by richkeenan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mfds) [reply README.md:26]: damn, i'm dumb LOL

Comment (mfds) [reply README.md:26]: need to fix docker-go as I did the same there 🤦

Comment (chriskurzeja) [reply README.md:26]: Easily missed :)

Comment (mfds) [reply README.md:26]: Thanks Chris

**Review by chriskurzeja — APPROVED** [review]

**Pull request merged**